receiver/vcenterreceiver/metrics.go

```diff
@@ -151,12 +151,17 @@ var hostPerfMetricList = []string{
 	"net.usage.average",
 	"net.errorsRx.summation",
 	"net.errorsTx.summation",
+	"net.droppedTx.summation",
+	"net.droppedRx.summation",
 	// disk metrics
 	"disk.totalReadLatency.average",
 	"disk.totalWriteLatency.average",
 	"disk.maxTotalLatency.latest",
 	"disk.read.average",
 	"disk.write.average",
+	// cpu metrics
+	"cpu.reservedCapacity.average",
+	"cpu.totalCapacity.average",
```

Comment:
Do you know how different this number ends up (if at all) from `numCpuCores` * `cpuMhz`?

Reply:
👆

Reply:
Yea, so I've been trying to figure this out. From what I understand `totalCapacity` would be the more accurate metric to measure here. In most cases the numbers should be very close together (usually a bit lower). This is because numCpuCores might get artificially inflated by any logical cores caused by hyper threading. Beyond that the numbers should be very similar if not the same

Reply:
After talking with Stefan a bit more on this topic, we uncovered that what we think of as totalCapacity as shown on these performance metrics is a bit different than what someone would normally think (numCpuCores * cpuMhz) as shown on the vSphere client.
 The difference being the calculated total capacity using the cpuCores and cpuMhz would be talking about how much capacity the host has for it 
<img width="342" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/38274348/2b726207-d48c-45dc-91cf-f1edb62614ba">
The performance metric totalCapacity is referring to the Total reservation capacity
<img width="349" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/38274348/057b097b-0144-4326-bf35-a668dc277386">
and the performance metric reservedCapacity is referring "used reservation" of the total reservation capacity.
<img width="367" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/38274348/c7dfcfc1-bee2-43ae-bea7-d4e16e36c060">

<img width="820" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/38274348/d9f25949-99f2-409f-903d-2597d6a9a7a8">

In the end, we think both the performance metric and the quickstat metric are both very useful depending on the user use case and setup, so we will be including them both.